src/pykeen/hpo/hpo.py

```diff
@@ -926,7 +926,7 @@ def _set_study_dataset(
             raise ValueError("Cannot specify dataset and training, testing and validation")
         elif isinstance(dataset, (str, pathlib.Path)):
             if isinstance(dataset, str) and has_dataset(dataset):
-                study.set_user_attr("dataset", get_dataset(dataset=dataset).get_normalized_name())
+                study.set_user_attr("dataset", dataset_resolver.normalize(dataset))
```

Comment:
should use get_normalized_name in case of caching

Reply:
d073e4a